packages/site/pages/install.js

```diff
@@ -25,7 +25,13 @@ Use of the Core or Components requires the existence of the Design System normal
 npm install @pluralsight/ps-design-system-normalize
 \`\`\`
 
-Include this vanilla CSS in your application in a method appropriate for your project. For usage options, follow the [Core Usage docs](/core/usage) patterns.
```

Comment:
Why remove this?  Still seems valid (ie, all components still require normalize)

Reply:
oops